tests/pages/index.test.tsx

```diff
@@ -15,7 +15,7 @@ jest.mock('next/router', () => ({
 let scenarioContent: ScenarioContent
 
 beforeAll(() => {
-  scenarioContent = getScenarioContent(apiGatewayStub(ScenarioType.Scenario1))
+  scenarioContent = getScenarioContent(apiGatewayStub(ScenarioType.Scenario2))
```

Comment:
Happy to switch this back to Scenario1 if there's a better quick way to output a scenario with sub-bullets in a snapshot